x/incentives/keeper/distribute_test.go

```diff
@@ -177,80 +185,107 @@ func (suite *KeeperTestSuite) TestSyntheticDistribute() {
 
 }
 
-// TestGetModuleToDistributeCoins tests the sum of coins yet to be distributed for all of the module is correct.
-func (suite *KeeperTestSuite) TestGetModuleToDistributeCoins() {
-	suite.SetupTest()
+// TestGetModuleToDistributeAndDistributedCoins tests the sum of coins yet to be distributed
+// and the distributed coins is correct.
+func (suite *KeeperTestSuite) TestGetModuleToDistributeAndDistributedCoins() {
+	lockOwner := sdk.AccAddress([]byte("addr1---------------"))
 
-	// check that the sum of coins yet to be distributed is nil
-	coins := suite.App.IncentivesKeeper.GetModuleToDistributeCoins(suite.Ctx)
-	suite.Require().Equal(coins, sdk.Coins(nil))
+	tests := []struct {
+		// each sdk.Coins in initialGaugeCoins will be used to create a new gauge
+		initialGaugeCoins []sdk.Coins
+		// sdk.Coins to add to gauge with gauge id
+		coinsToAddToGauges map[uint64]sdk.Coins
+	}{
```

Comment:
Should we add locks to make as a field here, and expected token amounts in addr by epoch num?

Reply:
this test only takes into account of total coins distributed and to be distributed, should we also check for reward per accounts. If so I think we should do that in a different test.